rebasehelper/application.py

```diff
@@ -217,12 +217,16 @@ def _initialize_data(self):
         self.old_rest_sources = [os.path.abspath(x) for x in self.spec_file.get_sources()[1:]]
         self.new_rest_sources = [os.path.abspath(x) for x in self.rebase_spec_file.get_sources()[1:]]
 
-    def get_rpm_packages(self, dirname):
-        """
-        Function returns RPM packages stored in dirname/old and dirname/new directories
+    def get_rpm_packages(self, dirname: str) -> bool:
+        """Function returns RPM packages stored in dirname/old-build
+        and dirname/new-build directories.
+
+        Args:
+            dirname: Path to a directory with RPMs.
+
+        Returns:
+            Whether any packages were found.
```

Comment:
This is in contradiction with the description, which is plain wrong anyway and needs to be replaced.